### PR TITLE
Adds ASMAT3 cruise to cross-mio

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

**Comment by cdobs** [comment]
Cruise information can be reviewed in the QLR (3213-00007) and information about the vessel can be found here: https://nicholas.duke.edu/marinelab/about/marine-operations

File: cruise/CruiseInformation.csv [hunk 1]
```diff
@@ -323,6 +323,8 @@ EK20230608,Elakha,2023-06-08T15:49:00,2023-06-08T22:08:00,Endurance glider recov
 EK20230615,Elakha,2023-06-15T15:32:00,2023-06-15T23:45:00,Endurance glider deployments
 EK20230616,Elakha,2023-06-16T15:35:00,2023-06-16T20:55:00,Endurance glider and CSPP deployments
 SP20230711,Storm Petrel,2023-07-11T14:00:00,2023-07-12T23:45:00,"Endurance ROV ops, CE06ISSP-17 deployment, and glider operations"
+SW20230724,Shearwater,2023-07-24T14:00:00,2023-07-25T12:00:00,ASMAT3 test AUV deployments
```
Comment (rtravis-whoi) [inline cruise/CruiseInformation.csv:326]: good

File: cruise/CruiseInformation.csv [hunk 2]
```diff
@@ -323,6 +323,8 @@ EK20230608,Elakha,2023-06-08T15:49:00,2023-06-08T22:08:00,Endurance glider recov
...
 EK20230728,Elakha,2023-07-28T17:00:00,2023-07-28T21:45:00,Endurance ce02shsp-00032 CSPP deployment
+VA20230731,Virginia,2023-07-31T20:00:00,2023-08-02T07:30,Pioneer AST3a AS03CPPM-00001 recovery
```
Comment (rtravis-whoi) [inline cruise/CruiseInformation.csv:328]: good

File: vessel/shiplist.csv [hunk 3]
```diff
@@ -59,11 +59,13 @@ SI,M/V,Scartlett Isabella,,BHC,,,,,,,,Y,N
 STD,R/V,SeaTow Delmarva,,SeaTow,,,,,,,,Y,N
 STO,R/V,SeaTow Obsession,,SeaTow,,,,,,,,Y,N
 SKQ,R/V,Sikuliaq,33BI,UAF,WDG7520,338417000,9578945,80,7.3,5.7,,Y,Y
+SW,R/V,Shearwater,,Duke University Marine Lab,,,,23.5,12.4,1.8,,Y,Y
```
Comment (rtravis-whoi) [inline vessel/shiplist.csv:62]: good

File: vessel/shiplist.csv [hunk 4]
```diff
@@ -59,11 +59,13 @@ SI,M/V,Scartlett Isabella,,BHC,,,,,,,,Y,N
...
 TS,F/V,Tenacious II,,TM,,,,,,,,Y,N
 TN,R/V,Thomas G. Thompson,3250,UW,KTDQ,366345000,8814419,83.5,7.5,5.9,AGOR-23,Y,Y
 WT,R/V,Thomas Washington,31WT,SIO,KGWU,,,63.7,13.8,4.9,AGOR-10,(N),Y
 TA,R/V,Tioga,,WHOI,,,,,,,,Y,N
 V,R/V,Vema,31VE,LDEO,J8RU,,7738383,,,,,(N),Y
+VA,R/V,Virginia,,Virginia Institute of Marine Science,WDK4341,368054210,,28.3,5.7,2.9,,Y,Y
```
Comment (rtravis-whoi) [inline vessel/shiplist.csv:68]: good

**Review by rtravis-whoi — APPROVED** [review]

File: cruise/CruiseInformation.csv (hunk 2 above)
Comment (reedan88) [inline cruise/CruiseInformation.csv:328]: Looks good.

File: vessel/shiplist.csv (hunk 4 above)
Comment (reedan88) [inline vessel/shiplist.csv:68]: Looks good

**Review by reedan88 — APPROVED** [review]
Looks good.

**Pull request merged**